### PR TITLE
NAVAND-836: add RouteOptions#suppressVoiceInstructionLocalNames

### PR DESCRIPTION
**Pull request by dzinad**

### CODE REVIEW

**Comment by dzinad** [comment]
Added DO NOT MERGE as we're waiting for:
1) API fix release;
2) Valhalla support.

**Comment by dzinad** [comment]
Removed DO NOT MERGE as the fix was released on the NavAPI side and we are not waiting for Valhalla support.

**Review by Zayankovsky — APPROVED** [review]

**Review by Guardiola31337 — APPROVED** [review]

File: services-directions/src/main/java/com/mapbox/api/directions/v5/DirectionsService.java [hunk 1]
```diff
@@ -20,47 +20,48 @@ interface DirectionsService {
...
-   * @param snappingIncludeClosures       {@link RouteOptions#snappingIncludeClosures()}
-   * @param snappingIncludeStaticClosures {@link RouteOptions#snappingIncludeStaticClosures()}
-   * @param arriveBy                      {@link RouteOptions#arriveBy()}
-   * @param departAt                      {@link RouteOptions#departAt()}
-   * @param maxHeight                     {@link RouteOptions#maxHeight()}
-   * @param maxWidth                      {@link RouteOptions#maxWidth()}
-   * @param computeTollCost               {@link RouteOptions#computeTollCost()}
-   * @param waypointsPerRoute             {@link RouteOptions#waypointsPerRoute()}
-   * @param metadata                      {@link RouteOptions#metadata()}
+   * @param userAgent                          the user agent
```
Comment (Guardiola31337) [inline services-directions/src/main/java/com/mapbox/api/directions/v5/DirectionsService.java:23]: Do we need to change the formatting? 🤔

Comment (dzinad) [reply services-directions/src/main/java/com/mapbox/api/directions/v5/DirectionsService.java:23]: Oh, I added the parameter and then removed it. But the formatting stayed. I'll revert it.

**Comment by Zayankovsky** [comment]
How is this option supposed to work? When I set it to `true`, I still get voice instructions like `In 100 meters, Turn right onto вуліца Кірава.` when driving in Minsk with English language. Amazon Polly pronounces such instructions as `In 100 meters, Turn right onto.`. Are we still waiting for backend support?

**Comment by dzinad** [comment]
> Are we still waiting for backend support?

Yes. More info is in the comments of https://mapbox.atlassian.net/browse/NAVAND-836.

**Pull request merged**